CAENHVAsynApp/src/channel.cpp

```diff
@@ -101,32 +101,39 @@ void IChannel::GetChannelParams()
 
         uint32_t type, mode;
 
-        if ( CAENHV_GetChParamProp(handle, slot, channel, p[i], "Type", &type) != CAENHV_OK )
-            throw std::runtime_error("CAENHV_GetChParamProp failed: " + std::string(CAENHV_GetError(handle)));
+        if ( CAENHV_GetChParamProp(handle, slot, channel, p[i], "Type", &type) != CAENHV_OK ) {
+            std::cerr << "CAENHV_GetChParamProp failed: " << std::string(CAENHV_GetError(handle)) << std::endl;
+            continue;
```

Comment:
error message should include `p[i]`

Reply:
Also std::string is unnecessary around `CAENHV_GetError` if you are using `std::cerr`, it was only needed for the previous string concatenation.